### PR TITLE
Added doc for days_since

### PR DESCRIPTION
**Pull request by charleso**

### CODE REVIEW

File: doc/design/feature-gen.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ be summed.
 ### `count_unique`
 Return the number of unique fact values in `facts`.
 
+### `days_since`
+Return the number of days _after_ a date _value_ and the `ref`.
```
Comment (FrontierOtr) [inline doc/design/feature-gen.md]: After doesn't make sense here, you're implying they're the same day or it's after both of them. It returns the number of days between a date value and the reference.

**Comment by charleso** [comment]
@FrontierOtr Good point. Better?

File: doc/design/feature-gen.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ be summed.
 ### `count_unique`
 Return the number of unique fact values in `facts`.
 
+### `days_since`
+Return the number of days between a date _value_ and the `ref`.
+If the value is _after_ the `ref` then the value will be negative.
```
Comment (FrontierOtr) [inline doc/design/feature-gen.md]: The result will be negative.

**Comment by nhibberd** [comment]
@charleso @HuwCampbell bump

**Pull request merged**